src/main/java/manager/InMemoryTaskManager.java

```diff
@@ -1,32 +1,38 @@
-package controllers;
+package manager;
 
-import java.util.*;
+import task.Epic;
+import task.Managers;
+import task.Subtask;
+import task.Task;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
 
-import model.*;
-import org.junit.jupiter.api.extension.ExtendWith;
 
 public class InMemoryTaskManager implements ITaskManager {
-    private  HistoryManager historyManager = null;
-    private HashMap<Integer, Epic> epics = new HashMap<>();
-    private HashMap<Integer, Task> tasks = new HashMap<>();
-    private HashMap<Integer,Subtask> subtasks = new HashMap<>();
+    private HistoryManager historyManager = null;
+    private final HashMap<Integer, Epic> epics = new HashMap<>();
+    private final HashMap<Integer, Task> tasks = new HashMap<>();
+    private final HashMap<Integer, Subtask> subtasks = new HashMap<>();
```

Comment:
Поправь и для мап выше
```suggestion
    private final Map<Integer, Subtask> subtasks = new HashMap<>();
```